Resources/Textures/Objects/Specific/Research/anomalyscanner.rsi/meta.json

```diff
@@ -5,10 +5,18 @@
     "y": 32
   },
   "license": "CC0-1.0",
-  "copyright": "Created by EmoGarbage",
+  "copyright": "Created by EmoGarbage, Inhands by TiniestShark based off of /tg station's health analyzer",
```

Comment:
You need to actually include a tg commit link wherein the sprite is contained. Refer to other meta.json files for this.
You can optionally track down the exact sprite location since this is a single sprite, but that is not nessacarry.

Reply:
I linked the sprite file directly, it's a sheet that includes all the medical inhands since it seems they don't use individual files. Does that work all right?

Reply:
You need to actually link a commit the sprite is in - the tg repo could update this sprite at any time.
Additionally, you should probably relicense the RSI to CC-BY-SA-3.0, since it's using tgstation assets. I'm not sure the anomaly scanner even classifies as being CC0 considering it's a medical scanner edit, but maybe it's different enough. I'm pretty sure it's fine to do this, at least.

Additionally... I apologize for this, but I was actually wrong about directly linking sprites here. You can't directly link inhands without bloating the RSI, as they're seperated in to two images. You should just use a commit that contains the sprite in it, like most other meta.jsons do.

Reply:
> You need to actually link a commit the sprite is in - the tg repo could update this sprite at any time.
> Additionally, you should probably relicense the RSI to CC-BY-SA-3.0, since it's using tgstation assets. I'm not sure the anomaly scanner even classifies as being CC0 considering it's a medical scanner edit, but maybe it's different enough. I'm pretty sure it's fine to do this, at least.
> 
> Additionally... I apologize for this, but I was actually wrong about directly linking sprites here. You can't directly link inhands without bloating the RSI, as they're seperated in to two images. You should just use a commit that contains the sprite in it, like most other meta.jsons do.

hi, is the license written correctly in my pull requests?
for example, I drew in-hands sprites for the barber scissors, do I need to change anything in the license?

Reply:
All right, updated the license and linked a commit the sprite was contained in. I should probably look up the differences in copyright licenses for future reference.